saleor/graphql/order/tests/benchmark/fixtures.py

```diff
@@ -47,6 +48,23 @@ def _prepare_events_for_order(order):
     ]
 
 
+def _prepare_lines_for_order(order, variant_with_image):
+    price = TaxedMoney(
+        net=Money(amount=5, currency="USD"), gross=Money(amount=5, currency="USD")
+    )
+    return [
+        OrderLine(
+            order=order,
+            variant=variant_with_image,
+            quantity=5,
+            is_shipping_required=False,
```

Comment:
Not sure if I fully understand the case but shouldn't we set here `is_shipping_required=True` ? And add shipping method for the orders?

Reply:
The point was to create thumbnails, those required lines with medias and lines have `is_shipping_required` as necessary. But you're right this can be quite confusing, as we create an order which would not be created by Saleor in any way (it's a bug if it would).